source/reference/methods.txt

```diff
@@ -730,6 +730,233 @@ Query Plan Cache Methods
        :manual:`plan cache entries </core/query-plans/>` 
        for a collection.
 
+Replication Methods 
+-------------------
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Method
+
+     - Description
+
+   * - :method:`rs.add()`
+
+     - Adds a member to the replica set. You must connect to the 
+       primary of the replica set to run this method.
+
+   * - :method:`rs.addArb()`
+
+     - Adds an arbiter to an existing replica set.
+
+   * - :method:`rs.config()`
+
+     - Returns a document that contains the current replica set configuration.
+
+   * - :method:`rs.freeze()`
+
+     - Makes the replica set member that ``mongosh`` is connected to 
+       ineligible to become primary for the specified duration. The 
+       duration must be specified in seconds.
+
+   * - :method:`db.getReplicationInfo()`
+
+     - Returns the status of the replica set from the oplog data.
+
+   * - :method:`rs.initiate()`
+
+     - Initializes a new replica set.
+
+   * - ``rs.isMaster``
+
+     - Returns the replica set configuration, role, and status for 
+       the ``mongod`` instance that ``mongosh`` is connected to. 
+       
+       The ``mongosh`` ``rs.isMaster()`` method wraps the server :dbcommand:`isMaster` command.
+
+   * - :method:`db.printReplicationInfo()`
+
+     - Returns the oplog of the replica set member that ``mongosh`` is 
+       connected to. 
+       
+       This is identical to the :method:`rs.printReplicationInfo()` method.
```

Comment:
These statements are not used consistently: `This is identical to ...`. If the same applies for methods listed above and below, consider adding these phrases each of those.

An alternative option is to remove these statements. I can see readers viewing them as redundant.

Reply:
I didn't actually write up these particular methods (just moved them around to put them in alphabetical order) so I'm not sure what the original intent was, but it seems reasonable to remove the "identical" sentences.